modules/gltf/extensions/physics/gltf_document_extension_physics.cpp

```diff
@@ -34,13 +34,26 @@
 
 // Import process.
 Error GLTFDocumentExtensionPhysics::import_preflight(Ref<GLTFState> p_state, Vector<String> p_extensions) {
-	if (!p_extensions.has("OMI_collider") && !p_extensions.has("OMI_physics_body")) {
+	if (!p_extensions.has("OMI_collider") && !p_extensions.has("OMI_physics_body") && !p_extensions.has("OMI_physics_shape")) {
 		return ERR_SKIP;
 	}
 	Dictionary state_json = p_state->get_json();
 	if (state_json.has("extensions")) {
 		Dictionary state_extensions = state_json["extensions"];
-		if (state_extensions.has("OMI_collider")) {
+		if (state_extensions.has("OMI_physics_shape")) {
+			Dictionary omi_physics_shape_ext = state_extensions["OMI_physics_shape"];
+			if (omi_physics_shape_ext.has("shapes")) {
+				Array state_shape_dicts = omi_physics_shape_ext["shapes"];
+				if (state_shape_dicts.size() > 0) {
+					Array state_shapes;
+					for (int i = 0; i < state_shape_dicts.size(); i++) {
+						state_shapes.push_back(GLTFPhysicsShape::from_dictionary(state_shape_dicts[i]));
+					}
+					p_state->set_additional_data("GLTFPhysicsShapes", state_shapes);
```

Comment:
just a comment but some use StringName / SNAME while others don't... Also it's a little hard to keep track of these additional data keys to make sure there isn't a typo / mistake e.g. plural vs not

Reply:
The difference in plurality is intentional to make it easier to search for unique keys with Ctrl+F. Keys that differ only in plurality will not appear on the same object.